### ISSUE TITLE
[Bug]: Constraint error when opening the share panel on iOS

### ISSUE THREAD

**Issue by MaximeDelhorme** [fluttercommunity/plus_plugins#1440]
### Platform

iPhone 15.7.1

### Plugin

share_plus

### Version

6.3.0

### Flutter SDK

2.18.0

### Steps to reproduce

1. Execute the code joined in the next section
2. Tap on the big red square in the middle of the screen
3. Error appears in IDE log

### Code Sample

```dart
import 'package:flutter/material.dart';
import 'package:flutter/rendering.dart';
import 'dart:io';
import 'dart:typed_data';
import 'dart:ui' as ui;
import 'package:path_provider/path_provider.dart';
import 'package:share_plus/share_plus.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: const MyHomePage(),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({
    super.key,
  });

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  GlobalKey? shareKey;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
          child: GestureDetector(
        onTap: () async {
          final image = await capture(shareKey!);
          shareImage(image, context);
        },
        child: SizedBox(
          width: 175.0,
          height: 175.0,
          child: WidgetToImage(builder: (key) {
            this.shareKey = key;

            return Container(
              color: Colors.red,
            );
          }),
        ),
      )),
    );
  }

  Future capture(GlobalKey key) async {
    RenderRepaintBoundary boundary =
        key.currentContext!.findRenderObject() as RenderRepaintBoundary;
    final image = await boundary.toImage(pixelRatio: 4);
    final byteData = await image.toByteData(format: ui.ImageByteFormat.png);
    final pngBytes = byteData!.buffer.asUint8List();

    return pngBytes;
  }

  void shareImage(Uint8List image, BuildContext context) async {
    final tempDir = await getTemporaryDirectory();
    final file = await File('${tempDir.path}/image.jpg').create();
    file.writeAsBytesSync(image);
    final files = <XFile>[XFile('${tempDir.path}/image.jpg')];
    final box = context.findRenderObject() as RenderBox?;

    ShareResult results = await Share.shareXFiles(files,
        sharePositionOrigin: box!.localToGlobal(Offset.zero) & box.size);
  }
}

class WidgetToImage extends StatefulWidget {
  final Function(GlobalKey key) builder;

  const WidgetToImage({
    required this.builder,
    Key? key,
  }) : super(key: key);

  @override
  WidgetToImageState createState() => WidgetToImageState();
}

class WidgetToImageState extends State<WidgetToImage> {
  final globalKey = GlobalKey();

  @override
  Widget build(BuildContext context) {
    return RepaintBoundary(
      key: globalKey,
      child: widget.builder(globalKey),
    );
  }
}
```

### Logs

```shell
[+15821 ms] [Default] Task Runner[39708]/1#3 LF=0 couldn't find entitlement CopresenceCore.Entitlement.publicAPI error nil
[ +739 ms] [Default] Task Runner[39708]/1#3 LF=0 couldn't find entitlement CopresenceCore.Entitlement.publicAPI error nil
[ +249 ms] [default] LaunchServices: store (null) or url (null) was nil: Error Domain=NSOSStatusErrorDomain Code=-54 "process may not map database" UserInfo={NSDebugDescription=process may not map database, _LSLine=264,
_LSFunction=-[_LSDReadClient getServerStoreWithCompletionHandler:]}
[   +1 ms] [default] Attempt to map database failed: permission was denied. This attempt will not be retried.
[        ] [db] Failed to initialize client context with error Error Domain=NSOSStatusErrorDomain Code=-54 "process may not map database" UserInfo={NSDebugDescription=process may not map database, _LSLine=264,
_LSFunction=-[_LSDReadClient getServerStoreWithCompletionHandler:]}
[  +36 ms] [default] -imageForImageDescriptor: can do IO please adopt -imageForDescriptor: for IO free drawing or -prepareImageForDescriptor: if IO is allowed. (This will become a fault soon.)
[   +8 ms] [default] LaunchServices: store (null) or url (null) was nil: Error Domain=NSOSStatusErrorDomain Code=-54 "process may not map database" UserInfo={NSDebugDescription=process may not map database, _LSLine=264,
_LSFunction=-[_LSDReadClient getServerStoreWithCompletionHandler:]}
[        ] [default] Attempt to map database failed: permission was denied. This attempt will not be retried.
[        ] [db] Failed to initialize client context with error Error Domain=NSOSStatusErrorDomain Code=-54 "process may not map database" UserInfo={NSDebugDescription=process may not map database, _LSLine=264,
_LSFunction=-[_LSDReadClient getServerStoreWithCompletionHandler:]}
[        ] [default] -imageForImageDescriptor: can do IO please adopt -imageForDescriptor: for IO free drawing or -prepareImageForDescriptor: if IO is allowed. (This will become a fault soon.)
[  +22 ms] [LayoutConstraints] Changing the translatesAutoresizingMaskIntoConstraints property of a UICollectionReusableView that is managed by a UICollectionView is not supported, and will result in incorrect self-sizing. View:
<_UIActivityContentFooterView: 0x11ec0dd00; baseClass = UICollectionReusableView; frame = (16 621; 343 52); layer = <CALayer: 0x283191be0>>
[+2647 ms] [LayoutConstraints] Unable to simultaneously satisfy constraints.
[        ] 	Probably at least one of the constraints in the following list is one you don't want. 
[        ] 	Try this: 
[        ] 		(1) look at each constraint and try to figure out which you don't expect; 
[        ] 		(2) find the code that added the unwanted constraint or constraints and fix it. 
[        ] (
[        ]     "<NSLayoutConstraint:0x2812adbd0 _UIActivityActionCellTitleLabel:0x1052f5b50.height >= 22   (active)>",
[        ]     "<NSLayoutConstraint:0x2812ad680 UIView:0x1052f4c70.height >= _UIActivityActionCellTitleLabel:0x1052f5b50.height + 30   (active)>",
[        ]     "<NSLayoutConstraint:0x2812a53b0 'UIView-Encapsulated-Layout-Height' UIView:0x1052f4c70.height == 30   (active)>"
[        ] )
[        ] Will attempt to recover by breaking constraint 
[        ] <NSLayoutConstraint:0x2812adbd0 _UIActivityActionCellTitleLabel:0x1052f5b50.height >= 22   (active)>
[        ] Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
[        ] The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
[  +30 ms] [LayoutConstraints] Unable to simultaneously satisfy constraints.
[        ] 	Probably at least one of the constraints in the following list is one you don't want. 
[        ] 	Try this: 
[        ] 		(1) look at each constraint and try to figure out which you don't expect; 
[        ] 		(2) find the code that added the unwanted constraint or constraints and fix it. 
[        ] (
[        ]     "<NSLayoutConstraint:0x2812bf610 _UIActivityActionCellTitleLabel:0x1052d10d0.height >= 22   (active)>",
[        ]     "<NSLayoutConstraint:0x2812bf0c0 UIView:0x1052d01f0.height >= _UIActivityActionCellTitleLabel:0x1052d10d0.height + 30   (active)>",
[        ]     "<NSLayoutConstraint:0x2812a4d70 'UIView-Encapsulated-Layout-Height' UIView:0x1052d01f0.height == 30   (active)>"
[        ] )
[        ] Will attempt to recover by breaking constraint 
[        ] <NSLayoutConstraint:0x2812bf610 _UIActivityActionCellTitleLabel:0x1052d10d0.height >= 22   (active)>
[        ] Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
[        ] The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
[   +1 ms] [LayoutConstraints] Unable to simultaneously satisfy constraints.
[        ] 	Probably at least one of the constraints in the following list is one you don't want. 
[        ] 	Try this: 
[        ] 		(1) look at each constraint and try to figure out which you don't expect; 
[        ] 		(2) find the code that added the unwanted constraint or constraints and fix it. 
[        ] (
[        ]     "<NSLayoutConstraint:0x2812bc0f0 _UIActivityActionCellTitleLabel:0x1052c4da0.height >= 22   (active)>",
[        ]     "<NSLayoutConstraint:0x2812c7c50 UIView:0x1052c3ec0.height >= _UIActivityActionCellTitleLabel:0x1052c4da0.height + 30   (active)>",
[        ]     "<NSLayoutConstraint:0x2812a4a00 'UIView-Encapsulated-Layout-Height' UIView:0x1052c3ec0.height == 30   (active)>"
[        ] )
[        ] Will attempt to recover by breaking constraint 
[        ] <NSLayoutConstraint:0x2812bc0f0 _UIActivityActionCellTitleLabel:0x1052c4da0.height >= 22   (active)>
[        ] Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
[        ] The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
[        ] [LayoutConstraints] Unable to simultaneously satisfy constraints.
[        ] 	Probably at least one of the constraints in the following list is one you don't want. 
[        ] 	Try this: 
[        ] 		(1) look at each constraint and try to figure out which you don't expect; 
[        ] 		(2) find the code that added the unwanted constraint or constraints and fix it. 
[        ] (
[        ]     "<NSLayoutConstraint:0x2812bab20 _UIActivityActionCellTitleLabel:0x1052db8c0.height >= 22   (active)>",
[        ]     "<NSLayoutConstraint:0x2812ba5d0 UIView:0x1052da9e0.height >= _UIActivityActionCellTitleLabel:0x1052db8c0.height + 30   (active)>",
[        ]     "<NSLayoutConstraint:0x2812a4ff0 'UIView-Encapsulated-Layout-Height' UIView:0x1052da9e0.height == 30   (active)>"
[        ] )
[        ] Will attempt to recover by breaking constraint 
[        ] <NSLayoutConstraint:0x2812bab20 _UIActivityActionCellTitleLabel:0x1052db8c0.height >= 22   (active)>
[        ] Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
[        ] The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
[        ] [LayoutConstraints] Unable to simultaneously satisfy constraints.
[        ] 	Probably at least one of the constraints in the following list is one you don't want. 
[        ] 	Try this: 
[        ] 		(1) look at each constraint and try to figure out which you don't expect; 
[        ] 		(2) find the code that added the unwanted constraint or constraints and fix it. 
[        ] (
[        ]     "<NSLayoutConstraint:0x2812b4230 _UIActivityActionCellTitleLabel:0x1052d5b50.height >= 22   (active)>",
[        ]     "<NSLayoutConstraint:0x2812c3d40 UIView:0x1052d4c70.height >= _UIActivityActionCellTitleLabel:0x1052d5b50.height + 30   (active)>",
[        ]     "<NSLayoutConstraint:0x2812a4eb0 'UIView-Encapsulated-Layout-Height' UIView:0x1052d4c70.height == 30   (active)>"
[        ] )
[        ] Will attempt to recover by breaking constraint 
[        ] <NSLayoutConstraint:0x2812b4230 _UIActivityActionCellTitleLabel:0x1052d5b50.height >= 22   (active)>
[        ] Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
[        ] The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
[        ] [LayoutConstraints] Unable to simultaneously satisfy constraints.
[        ] 	Probably at least one of the constraints in the following list is one you don't want. 
[        ] 	Try this: 
[        ] 		(1) look at each constraint and try to figure out which you don't expect; 
[        ] 		(2) find the code that added the unwanted constraint or constraints and fix it. 
[        ] (
[        ]     "<NSLayoutConstraint:0x2812a41e0 _UIActivityActionCellTitleLabel:0x105319430.height >= 22   (active)>",
[        ]     "<NSLayoutConstraint:0x2812cc960 UIView:0x105318550.height >= _UIActivityActionCellTitleLabel:0x105319430.height + 30   (active)>",
[        ]     "<NSLayoutConstraint:0x2812a5270 'UIView-Encapsulated-Layout-Height' UIView:0x105318550.height == 30   (active)>"
[        ] )
[        ] Will attempt to recover by breaking constraint 
[        ] <NSLayoutConstraint:0x2812a41e0 _UIActivityActionCellTitleLabel:0x105319430.height >= 22   (active)>
[        ] Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
[        ] The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
[+2727 ms] [ShareSheet] connection invalidated
```

### Flutter Doctor

```shell
✓] Flutter (Channel stable, 3.3.2, on macOS 12.5 21G72 darwin-arm, locale en-FR)
    • Flutter version 3.3.2 on channel stable at /Users/maxime/Documents/Development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision e3c29ec00c (4 months ago), 2022-09-14 08:46:55 -0500
    • Engine revision a4ff2c53d8
    • Dart version 2.18.1
    • DevTools version 2.15.0

[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at /Users/maxime/Library/Android/sdk
    • Platform android-33, build-tools 32.0.0
    • ANDROID_HOME = /Users/maxime/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14C18
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] Connected device (3 available)
    • Maxime’s iPhone (mobile) • c1eaf4d4f86b91ff0d97f81bacb95f8fe596ef4d • ios            • iOS 15.7.1 19H117
    • macOS (desktop)          • macos                                    • darwin-arm64   • macOS 12.5 21G72 darwin-arm
    • Chrome (web)             • chrome                                   • web-javascript • Google Chrome 108.0.5359.124

[✓] HTTP Host Availability
    • All required HTTP hosts are available
```

### Checklist before submitting a bug

- [X] I Google'd a solution and I couldn't find it
- [X] I searched on StackOverflow for a solution and I couldn't find it
- [X] I read the README.md file of the plugin
- [X] I'm using the latest version of the plugin
- [X] All dependencies are up to date with `flutter pub upgrade`
- [X] I did a `flutter clean`
- [X] I tried running the example project

**Comment by lflorent** [comment]
Same issue with a text sharing and version 6.3.0

```dart
Share.share(widget.articleData!.title! + " " + widget.articleData!.link!)
```

## Logs

```
[default] LaunchServices: store (null) or url (null) was nil: Error Domain=NSOSStatusErrorDomain Code=-54 "process may not map database" UserInfo={NSDebugDescription=process may not map database, _LSLine=66, _LSFunction=_LSServer_GetServerStoreForConnectionWithCompletionHandler}
[default] Attempt to map database failed: permission was denied. This attempt will not be retried.
[db] Failed to initialize client context with error Error Domain=NSOSStatusErrorDomain Code=-54 "process may not map database" UserInfo={NSDebugDescription=process may not map database, _LSLine=66, _LSFunction=_LSServer_GetServerStoreForConnectionWithCompletionHandler}
[default] -imageForImageDescriptor: can do IO please adopt -imageForDescriptor: for IO free drawing or -prepareImageForDescriptor: if IO is allowed. (This will become a fault soon.)
[LayoutConstraints] Changing the translatesAutoresizingMaskIntoConstraints property of a UICollectionReusableView that is managed by a UICollectionView is not supported, and will result in incorrect self-sizing. View: <_UIActivityContentFooterView: 0x128b0ba40; baseClass = UICollectionReusableView; frame = (16 517.667; 343 52); layer = <CALayer: 0x28102f120>>
[LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x2833d15e0 _UIActivityActionCellTitleLabel:0x115ff37b0.height >= 22   (active)>",
    "<NSLayoutConstraint:0x2833d1090 _UICollectionViewListCellContentView:0x115ff1f20.height >= _UIActivityActionCellTitleLabel:0x115ff37b0.height + 30   (active)>",
    "<NSLayoutConstraint:0x28332e580 'UIView-Encapsulated-Layout-Height' _UICollectionViewListCellContentView:0x115ff1f20.height == 30   (active)>"
)


Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x2833d15e0 _UIActivityActionCellTitleLabel:0x115ff37b0.height >= 22   (active)>


Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
[LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x28332c410 _UIActivityActionCellTitleLabel:0x128b05e10.height >= 22   (active)>",
    "<NSLayoutConstraint:0x2833d3f20 _UICollectionViewListCellContentView:0x128b04580.height >= _UIActivityActionCellTitleLabel:0x128b05e10.height + 30   (active)>",
    "<NSLayoutConstraint:0x28332e6c0 'UIView-Encapsulated-Layout-Height' _UICollectionViewListCellContentView:0x128b04580.height == 30   (active)>"
)


Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x28332c410 _UIActivityActionCellTitleLabel:0x128b05e10.height >= 22   (active)>


Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
[LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x2833d2d50 _UIActivityActionCellTitleLabel:0x115ffaab0.height >= 22   (active)>",
    "<NSLayoutConstraint:0x2833d2800 _UICollectionViewListCellContentView:0x115ff9220.height >= _UIActivityActionCellTitleLabel:0x115ffaab0.height + 30   (active)>",
    "<NSLayoutConstraint:0x28332e940 'UIView-Encapsulated-Layout-Height' _UICollectionViewListCellContentView:0x115ff9220.height == 30   (active)>"
)


Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x2833d2d50 _UIActivityActionCellTitleLabel:0x115ffaab0.height >= 22   (active)>


Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
```

This issue seems similar to previous ones like https://github.com/fluttercommunity/plus_plugins/issues/730 or https://github.com/fluttercommunity/plus_plugins/issues/880

**Comment by rhamnett** [comment]
Experiencing the same problem

**Comment by andrepura** [comment]
same here
```
2023-02-07 11:01:20.092061+0100 Runner[3870:1249005] [default] LaunchServices: store (null) or url (null) was nil: Error Domain=NSOSStatusErrorDomain Code=-54 "process may not map database" UserInfo={NSDebugDescription=process may not map database, _LSLine=66, _LSFunction=_LSServer_GetServerStoreForConnectionWithCompletionHandler}
2023-02-07 11:01:20.092098+0100 Runner[3870:1249005] [default] Attempt to map database failed: permission was denied. This attempt will not be retried.
2023-02-07 11:01:20.092123+0100 Runner[3870:1249005] [db] Failed to initialize client context with error Error Domain=NSOSStatusErrorDomain Code=-54 "process may not map database" UserInfo={NSDebugDescription=process may not map database, _LSLine=66, _LSFunction=_LSServer_GetServerStoreForConnectionWithCompletionHandler}
2023-02-07 11:01:20.099214+0100 Runner[3870:1248763] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
	(Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSAutoresizingMaskLayoutConstraint:0x280a66df0 h=-&- v=--& _UIActivityContentNavigationBar:0x10c8accb0.minY == 0   (active, names: '|':UILayoutContainerView:0x10c8acae0 )>",
    "<NSAutoresizingMaskLayoutConstraint:0x280a66da0 h=-&- v=--& _UIActivityContentNavigationBar:0x10c8accb0.height == 50   (active)>",
    "<NSLayoutConstraint:0x280a9bd40 V:[_UIActivityContentNavigationBar:0x10c8accb0]-(0)-[UIFocusContainerGuide:0x28163cd20'UINavigationControllerContentFocusContainerGuide']   (active)>",
    "<NSLayoutConstraint:0x280a9be30 UIFocusContainerGuide:0x28163cd20'UINavigationControllerContentFocusContainerGuide'.bottom == UILayoutContainerView:0x10c8acae0.bottom   (active)>",
    "<NSLayoutConstraint:0x280a67750 V:|-(0)-[UILayoutContainerView:0x10c8acae0]   (active, names: '|':UIView:0x10c8cba90 )>",
    "<NSLayoutConstraint:0x280a67700 UILayoutContainerView:0x10c8acae0.bottom == UIView:0x10c8cba90.bottom   (active)>",
    "<NSLayoutConstraint:0x280a66440 'UIView-Encapsulated-Layout-Height' UIView:0x10c8cba90.height == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x280a9bd40 V:[_UIActivityContentNavigationBar:0x10c8accb0]-(0)-[UIFocusContainerGuide:0x28163cd20'UINavigationControllerContentFocusContainerGuide']   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
2023-02-07 11:01:20.133703+0100 Runner[3870:1248763] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x280a64280 H:|-(0)-[_UIActivityContentTitleView:0x10c8db520]   (active, names: '|':_UINavigationBarContentView:0x10c8a9890 )>",
    "<NSLayoutConstraint:0x280a642d0 _UIActivityContentTitleView:0x10c8db520.trailing == _UINavigationBarContentView:0x10c8a9890.trailing   (active)>",
    "<NSLayoutConstraint:0x280a653b0 LPLinkView:0x10c8beba0.leading == UILayoutGuide:0x28101ebc0'UIViewLayoutMarginsGuide'.leading   (active)>",
    "<NSLayoutConstraint:0x280a65630 LPLinkView:0x10c8beba0.trailing == UILayoutGuide:0x28101ebc0'UIViewLayoutMarginsGuide'.trailing   (active)>",
    "<NSLayoutConstraint:0x280a65c70 'UIView-Encapsulated-Layout-Width' _UINavigationBarContentView:0x10c8a9890.width == 6   (active)>",
    "<NSLayoutConstraint:0x280a64000 'UIView-leftMargin-guide-constraint' H:|-(29)-[UILayoutGuide:0x28101ebc0'UIViewLayoutMarginsGuide'](LTR)   (active, names: '|':_UIActivityContentTitleView:0x10c8db520 )>",
    "<NSLayoutConstraint:0x280a65040 'UIView-rightMargin-guide-constraint' H:[UILayoutGuide:0x28101ebc0'UIViewLayoutMarginsGuide']-(8)-|(LTR)   (active, names: '|':_UIActivityContentTitleView:0x10c8db520 )>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x280a65630 LPLinkView:0x10c8beba0.trailing == UILayoutGuide:0x28101ebc0'UIViewLayoutMarginsGuide'.trailing   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
2023-02-07 11:01:20.134005+0100 Runner[3870:1248763] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x280a9b250 UILayoutGuide:0x28101d340'BackButtonGuide(0x10c8a9bd0)'.leading == UILayoutGuide:0x28101eae0'UINavigationBarItemContentLayoutGuide'.leading   (active)>",
    "<NSLayoutConstraint:0x280a9b340 H:[UILayoutGuide:0x28101d340'BackButtonGuide(0x10c8a9bd0)']-(0)-[UILayoutGuide:0x28101ed80'LeadingBarGuide(0x10c8a9bd0)']   (active)>",
    "<NSLayoutConstraint:0x280a9b2f0 H:[UILayoutGuide:0x28101ed80'LeadingBarGuide(0x10c8a9bd0)']-(0)-[UILayoutGuide:0x28101e060'TitleViewGuide(0x10c8a9bd0)']   (active)>",
    "<NSLayoutConstraint:0x280a9b8e0 UILayoutGuide:0x28101e060'TitleViewGuide(0x10c8a9bd0)'.trailing <= UILayoutGuide:0x28101ea00'UIViewLayoutMarginsGuide'.trailing   (active)>",
    "<NSLayoutConstraint:0x280a99040 'UINavItemContentGuide-leading' H:|-(0)-[UILayoutGuide:0x28101eae0'UINavigationBarItemContentLayoutGuide']   (active, names: '|':_UINavigationBarContentView:0x10c8a9890 )>",
    "<NSLayoutConstraint:0x280a65c70 'UIView-Encapsulated-Layout-Width' _UINavigationBarContentView:0x10c8a9890.width == 6   (active)>",
    "<NSLayoutConstraint:0x280a9b7a0 'UIView-rightMargin-guide-constraint' H:[UILayoutGuide:0x28101ea00'UIViewLayoutMarginsGuide']-(16)-|(LTR)   (active, names: '|':_UINavigationBarContentView:0x10c8a9890 )>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x280a9b340 H:[UILayoutGuide:0x28101d340'BackButtonGuide(0x10c8a9bd0)']-(0)-[UILayoutGuide:0x28101ed80'LeadingBarGuide(0x10c8a9bd0)']   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
2023-02-07 11:01:20.134389+0100 Runner[3870:1248763] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x280a9b8e0 UILayoutGuide:0x28101e060'TitleViewGuide(0x10c8a9bd0)'.trailing <= UILayoutGuide:0x28101ea00'UIViewLayoutMarginsGuide'.trailing   (active)>",
    "<NSLayoutConstraint:0x280a9b890 UILayoutGuide:0x28101e060'TitleViewGuide(0x10c8a9bd0)'.leading >= UILayoutGuide:0x28101ea00'UIViewLayoutMarginsGuide'.leading   (active)>",
    "<NSLayoutConstraint:0x280a65c70 'UIView-Encapsulated-Layout-Width' _UINavigationBarContentView:0x10c8a9890.width == 6   (active)>",
    "<NSLayoutConstraint:0x280a9b700 'UIView-leftMargin-guide-constraint' H:|-(16)-[UILayoutGuide:0x28101ea00'UIViewLayoutMarginsGuide'](LTR)   (active, names: '|':_UINavigationBarContentView:0x10c8a9890 )>",
    "<NSLayoutConstraint:0x280a9b7a0 'UIView-rightMargin-guide-constraint' H:[UILayoutGuide:0x28101ea00'UIViewLayoutMarginsGuide']-(16)-|(LTR)   (active, names: '|':_UINavigationBarContentView:0x10c8a9890 )>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x280a9b8e0 UILayoutGuide:0x28101e060'TitleViewGuide(0x10c8a9bd0)'.trailing <= UILayoutGuide:0x28101ea00'UIViewLayoutMarginsGuide'.trailing   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
2023-02-07 11:01:20.143038+0100 Runner[3870:1248763] [UICollectionLayout] <_UIActivityCollectionViewCompositionalLayout 0x10c8889b0> ran into an error when computing the layout for section at index 0 in container <_UICollectionLayoutContainer: 0x283cd1c00 contentSize={-7, 0}; contentInsets={0, 0, 0, 0}}>. UICollectionView behavior with this layout is undefined. Break on UICollectionViewCompositionalLayoutAlertForInvalidLayout to catch this in the debugger.
Error: NSCollectionLayoutItem created with invalid combination of spacing and size specified. This group cannot fit even a single item. Inspect the spacing and size of the items in this group and ensure that they fit into the group when its effective size is {43.333333333333343, 128}.
Group: <NSCollectionLayoutGroup size={.absolute(43.3333), .estimated(128)};
	 edgeSpacing=<leading=.fixed(0); top=.fixed(0); trailing=.fixed(0); bottom=.fixed(0)>;
	 contentInsets={0, 0, 0, 0}>
	 group: subitems=
		 <NSCollectionLayoutItem size={.absolute(78), .estimated(128)};
		 edgeSpacing=<leading=.fixed(0); top=.fixed(0); trailing=.fixed(0); bottom=.fixed(0)>;
		 contentInsets={0, 0, 0, 0}>
	 interItemSpacing=.fixed(-95.33333333333333);
	 layoutDirection=.horizontal>
```

**Comment by jingluoguo** [comment]
https://github.com/fluttercommunity/plus_plugins/issues/870#issuecomment-1200419755
